matter_server/server/device_controller.py

```diff
@@ -281,10 +281,11 @@ async def commission_on_network(
                     filter=filter,
                 )
             else:
-                # Ip-adress provided, use CommissionIP method
+                ip_addr = self.server.scope_ipv6_lla(ip_addr)
                 LOGGER.info(
-                    "Starting Matter commissioning with IP using Node ID %s (attempt %s/%s).",
+                    "Starting Matter commissioning with IP using Node ID %s via IP address %s  (attempt %s/%s).",
```

Comment:
```suggestion
                    "Starting Matter commissioning with IP "
                    "using Node ID %s via IP address %s  (attempt %s/%s).",
```

Reply:
I've just updated that part, made the message bit shorter.